lib/inspec/plugin/v2/installer.rb

```diff
@@ -217,7 +217,10 @@ def validate_installation_opts(plugin_name, opts)
         if opts.key?(:version) && plugin_version_installed?(plugin_name, opts[:version])
           raise InstallError, "#{plugin_name} version #{opts[:version]} is already installed."
         else
-          raise InstallError, "#{plugin_name} is already installed. Use 'inspec plugin update' to change version."
+          # Do not redirect to plugin update when using gem based plugin
+          unless is_a_gem_based_plugin?(plugin_name)
```

Comment:
as discussed offline, this will be renamed to `gem_based_plugin?` and its implementation will accept `opts` and look for `:gem` keyword inside opts to determine if it is gem based plugin

Reply:
Making the change, yes!